### DISCUSSION TITLE
Add `removable-media` interface to snap

### DISCUSSION THREAD

**Discussion by isgj** [Kong/insomnia#3826] [Ideas]
Would you consider adding `removable-media` interface fro the snap build?
I wanted to import a file that sits on an external drive, but at the moment I can't.

**Comment by dimitropoulos** [comment]
I'm curious: do you (or anyone else) know whether flatpak supports this same functionality?

**Reply by isgj** [reply]
https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access

`--filesystem=host` seems to allow apps to use `/media` (at least in my case there are mounted the external drives)
